core/domain/question_services.py

```diff
@@ -201,3 +203,39 @@ def update_question(committer_id, question_id, change_list, commit_message):
     """
     question = apply_change_list(question_id, change_list)
     _save_question(committer_id, question, change_list, commit_message)
+
+
+def get_questions_batch(
+        collection_id, skill_ids, user_id, question_play_counts, batch_size):
+    """Fetches a batch of questions.
```

Comment:
Let's talk a bit more about how the args relate to the returned batch of questions. For example, from this docstring, I can't tell how changes in skill_ids, user_id, etc. would affect the returned batch.

Reply:
Updated.